### PR TITLE
Issue #ED-2577 fix: CLONE - Portal - Remove dependency with Sb-themes repo and testing inside portal

### PR DESCRIPTION
**Pull request by pradoshKumar**

### CODE REVIEW

File: src/app/client/src/app/modules/observation/components/observation-listing/observation-listing.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <app-landing-section [textToDisplay]="resourceService?.frmelmnts?.lbl?.observation" [svgToDisplay]="svgToDisplay"
```
Comment (sri-pusuluri) [inline src/app/client/src/app/modules/observation/components/observation-listing/observation-listing.component.html:1]: not required. you can remove

File: src/app/client/src/app/modules/groups/components/activity/activity-details/activity-details.component.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div class="sbt-center-container sbt-back-header sbt-mygroups-details relative9">
+<div class="mat-inside-page-container mat-app-background2 sbt-back-header sbt-mygroups-details relative9">
```
Comment (sri-pusuluri) [inline src/app/client/src/app/modules/groups/components/activity/activity-details/activity-details.component.html:1]: is this class requied " sbt-back-header "

Comment (pradoshKumar) [reply src/app/client/src/app/modules/groups/components/activity/activity-details/activity-details.component.html]: removed

File: src/app/client/src/app/modules/observation/components/observation-listing/observation-listing.component.html (hunk 1 above)
Comment (pradoshKumar) [reply src/app/client/src/app/modules/observation/components/observation-listing/observation-listing.component.html:1]: anyway we are not working on the observation module as per the suggestion provided by vinu kumar

**Pull request merged**